sections/synths/components/SynthPriceCol.tsx

```diff
@@ -3,23 +3,34 @@ import styled from 'styled-components';
 
 import media from 'styles/media';
 
-import { Period } from 'constants/period';
-
 import CurrencyPrice from 'components/Currency/CurrencyPrice';
 import { CurrencyKey } from '@synthetixio/contracts-interface';
 import useSelectedPriceCurrency from 'hooks/useSelectedPriceCurrency';
 import { NO_VALUE } from 'constants/placeholder';
-import useSynthetixQueries from '@synthetixio/queries';
+import useSynthetixQueries, { exchanges, SynthetixQueryContext } from '@synthetixio/queries';
+import React from 'react';
```

Comment:
Another one where you can just import `{useContext}`

Reply:
https://github.com/Synthetixio/staking/pull/769/commits/0a9be4a938eb698efd5a4b16244d182fdacbbbb5